pkg/ui/ccl/src/views/clusterviz/components/instructionsBox/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+
+import { LocalityTree } from "src/redux/localities";
+import { LocationTree } from "src/redux/locations";
+import { getConfigStatus } from "src/util/locations";
+import { pluralize } from "src/util/pluralize";
+import nodeMapScreenshot from "assets/nodeMapScreenshot.png";
+import "./instructionsBox.styl";
+
+interface InstructionsBoxProps {
+  localityTree: LocalityTree;
+  locationTree: LocationTree;
+}
+
+interface InstructionBoxState {
+  expanded: boolean;
+}
+
+const LOCAL_STORAGE_KEY = "clusterviz-instruction-box-expanded";
+
+export class InstructionsBox extends React.Component<InstructionsBoxProps, InstructionBoxState> {
+  constructor() {
+    super();
+    const valFromStorage = localStorage.getItem(LOCAL_STORAGE_KEY);
```

Comment:
it seems like it would be nice to have this use the same mechanism that the version dismissal uses, for consistency.  it doesn't look that complicated, take a look at `src/redux/alerts`